src/components/AgreementFilters/AgreementFilters.js

```diff
@@ -235,12 +238,32 @@ export default class AgreementFilters extends React.Component {
     />;
   }
 
+  renderStartDateFilter = () => {
+    return <DateFilter
+      activeFilters={this.props.activeFilters}
+      filterHandlers={this.props.filterHandlers}
+      name="startDate"
+    />;
+  }
+
+  renderEndDateFilter = () => {
+    return <DateFilter
+      activeFilters={this.props.activeFilters}
+      allowNull
```

Comment:
Lets name this `showNoEndDateCheckbox`

Reply:
But this could theoretically apply to either start or end date, for instance in Licenses. What about `showNoDateCheckbox`?

Reply:
Thinking through this again, I feel `hideNoEndDateCheckbox` would be better here, so that we can default the shared component to show the checkbox and hide it conditionaly. Also, please add a default prop value of `false` to `hideNoEndDateCheckbox`

Reply:
Have done both